### ISSUE TITLE
Blocking Amazon interstitial full-screen ad redirect when checking out

### ISSUE THREAD

**Issue by b1k3rdude** [einaregilsson/Redirector#390]
Evening

So I created a redirect but it dosent appears to work, but this could down to how to redirect is happening

from - https://www.amazon.co.uk/gp/buy/primeinterstitial/handlers/display.html
to - https://www.amazon.co.uk/gp/buy/spc/handlers/display.html

![redirfector](https://github.com/user-attachments/assets/55ec0f78-7d86-435b-841d-91b223cda803)
![trial_wtf](https://github.com/user-attachments/assets/98ff1bc2-4408-4620-a6a0-480e1135af73)

**Comment by jadedgnome** [comment]
try enabling historystate

**Comment by b1k3rdude** [comment]
@SmartManoj  sorry for the horrendously late reply.

No unfortunately, I have instead had to just block enough of the prime crap via ublock origin. And move away from Amazon for any purchases. A lot of the small things can be bought on ebay with little or no postage fees. And for the bigger items, I just had to remind myself there is and has always been more than enough other online & retail stores.

I like I imagine many, many others, are pretty much done with putting up with predatory & substandard retail behaviour and Amazon doing fcuk all about it including basic duty of care.

**Comment by Gitoffthelawn** [comment]
> [@SmartManoj](https://github.com/SmartManoj) sorry for the horrendously late reply.
> 
> No unfortunately, I have instead had to just block enough of the prime crap via ublock origin. And move away from Amazon for any purchases. A lot of the small things can be bought on ebay with little or no postage fees. And for the bigger items, I just had to remind myself there is and has always been more than enough other online & retail stores.
> 
> I like I imagine many, many others, are pretty much done with putting up with predatory & substandard retail behaviour and Amazon doing fcuk all about it including basic duty of care.

I agree 100%. 💯👍🏾  I am so done with Amazon.  The little bit of potential savings is never worth the big headaches and their refusal to be an honest business. (BTW, I am currently pursuing formal legal action against Amazon, and if others are interested, perhaps a class action is warranted.)
